dynawo/sources/Common/DYNTimer.cpp

```diff
@@ -49,8 +58,15 @@ Timers::add(const std::string& name, const double& time) {
 
 void
 Timers::add_(const std::string& name, const double& time) {
-  timers_[name] += time;
-  nbAppels_[name] += 1;
+#ifdef LANG_CXX11
+  std::stringstream ss;
+  ss << std::this_thread::get_id() << "_" << name;
+  std::string name_formatted = ss.str();
+#else
+  std::string name_formatted = name;
+#endif
+  timers_[name_formatted] += time;
+  nbAppels_[name_formatted] += 1;
```

Comment:
Was this tested? What does the output looks like (just by curiosity)?

Reply:
I didn't change this part so I didn't tested it

Reply:
I mean launching a SA in debug mode and taking a look at the output

Reply:
TIMER[140258212005952_evalJt_evalDerivatives] = 0.073168 secondes en 369 appels
TIMER[140258212005952_evalJt_initBusDerivatives] = 0.015188 secondes en 369 appels
TIMER[140258212005952_zzz filling] = 0.190309 secondes en 5968 appels
TIMER[140258212005952_zzz reading] = 0.575743 secondes en 5968 appels